spec/factories/upvs_factories.rb

```diff
@@ -4,4 +4,13 @@
     sequence(:uri) { |n| "ico://sk/#{10_000_000 + n}_90000" }
     sequence(:name) { |n| "Súkromná materská škola #{n}, Starozagorská 8, Košice" }
   end
+
+  factory :digital_service_and_form, class: Upvs::DigitalServiceAndForm do
```

Comment:
pozor, urcite treba prefixovat modulom kvoli koliziam v nazvoch

```suggestion
  factory :upvs_digital_service_and_form, class: Upvs::DigitalServiceAndForm do
```